src/pages/TrueSightV2/pages/SingleToken.tsx

```diff
@@ -201,19 +208,30 @@ const TokenDescription = ({ description }: { description: string }) => {
 
   return (
     <Row style={{ position: 'relative' }}>
-      <StyledTokenDescription
-        ref={ref}
-        show={show}
-        dangerouslySetInnerHTML={{
-          __html:
-            linkify(description) +
-            (isTextExceeded
-              ? `<span style="color:${
-                  theme.primary
-                }; cursor:pointer; margin-left:4px;" id="hide-token-description-span">${t`Hide`}</span>`
-              : ''),
-        }}
-      />
+      <StyledTokenDescription ref={ref} show={show}>
+        <span
+          dangerouslySetInnerHTML={{
+            __html: linkify(description),

```

Comment:
escapeScriptHtml(linkify(description)) 
or add escapeScriptHtml inside the function linkify